From 1e950b78eaa586357924c0ec767ecb040dd8a5ed Mon Sep 17 00:00:00 2001
From: svenate <salini.venate@intel.com>
Date: Tue, 15 Mar 2022 23:16:41 +0530
Subject: [PATCH] WA Remove -fno-strict-overflow to fix Android T build errors

This is a WA patch to resolve below build error:
clang: error: argument unused during compilation:
'-fno-strict-overflow' [-Werror,-Wunused-command-line-argument]

Signed-off-by: svenate <salini.venate@intel.com>
---
 src/Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/Makefile b/src/Makefile
index 86ed054..44ed0a2 100644
--- a/src/Makefile
+++ b/src/Makefile
@@ -33,7 +33,7 @@ CFLAGS += -Wall -Wextra -Wbad-function-cast \
 	  -Wcast-align -Wwrite-strings \
 	  -fno-delete-null-pointer-checks -fstack-protector-strong \
 	  -D_FORTIFY_SOURCE=2 \
-	  -fno-strict-overflow -fwrapv
+	  -fwrapv
 
 LDFLAGS = -lglib-2.0 -luuid -lmenu -lform -lncurses -ljson-c
 
-- 
2.17.1

